# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gst-plugins-good/libshout.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gst-plugins-good-0.10.3/ext/shout2/gstshout2.h.vanilla	2007-03-23 15:59:16.000000000 +0100
+++ gst-plugins-good-0.10.3/ext/shout2/gstshout2.h	2007-03-23 15:59:35.000000000 +0100
@@ -23,7 +23,7 @@
 
 #include <gst/gst.h>
 #include <gst/base/gstbasesink.h>
-#include <shout/shout.h>
+#include <shout.h>
 
 G_BEGIN_DECLS
 
